docs/source/handling_advanced_stypes/handle_text.rst

```diff
@@ -121,7 +124,7 @@ pre-computed embeddings, which can easily handle :obj:`~torch_frame.stype.text_e
     stype_encoder_dict = {
         stype.categorical: EmbeddingEncoder(),
         stype.numerical: LinearEncoder(),
-        stype.text_embedded: LinearEmbeddingEncoder()
+        stype.embedding: LinearEmbeddingEncoder()
```

Comment:
I might be misunderstanding something but open to any comments :) (cc @weihua916 @zechengz)

Reply:
We wouldn't know if the `TensorFrame` has `text_embedded` after materialization. We now raise an error with explanation on how to solve the issue.

Reply:
@akihironitta That's a good point. For now, I think we can be strict in `stype_encoder_dict` by only allowing parent stype. I think this should be fine since we already have a very informative error message. However, if our users do put child stype a lot in `stype_encoder_dict` (since that's indeed more intuitive), we may consider changing our design in the future.